src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -76,7 +76,7 @@ public function getRouteCollectionForRequest(Request $request)
     {
         $collection = new RouteCollection();
 
-        if ('' === $request->getRequestFormat()) {
+        if ('' === $request->getRequestFormat(null)) {
```

Comment:
there is a bug in older symfony versions that caches the default value, even if you pass a different default value in the next call. thes was fixed in `2.8.18`: https://github.com/symfony/http-foundation/commit/d45f940f3b55a0c99eae8482c726be264ad1c8dc